layouts/episodes/single.html

```diff
@@ -26,17 +26,19 @@ <h2 id="transcript">Transcript</h2>
 
```

Comment:
Same on line 24 above - extra `<p>` tag. (These are likely all old bugs that I introduced, btw. 🙃 )